src/app/public/modules/radio/radio.component.ts

```diff
@@ -145,9 +193,21 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._value;
   }
 
+/**
+ * Specifies an icon to display in place of the radio button. To group radio buttons like in
+ * the demo above, place the  * `sky-switch-icon-group` class on the direct parent element of the radio buttons.
```

Comment:
We have a stray `*` after `place the`. I assume it was part of the JSDoc setup that just got pulled into the description.

Reply:
Fixed